### DISCUSSION TITLE
Grundeinstellungen

### DISCUSSION THREAD

**Discussion by Michel83** [hoylabs/OpenDTU-OnBattery#704] [General]
Hi, 

bin jetzt auch erfolgreicher DTU on Battery Nutzer und total begeistert. Ich hatte knapp 3 Jahr ein ESmart3 womit ich eigenltich recht zufrieden war bis ich auf diese Projekt gestoßen bin.

Ich hätte eine Frage zur config meine Jetzige : 

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/36561525/d4c877c8-6d04-4288-bc56-b2a71eff73da)
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/36561525/820894c0-e82d-4898-9325-8e841cc6640a)

Im Prinzip habe ich erst mal nicht viel geändert, bei meiner Installation ist es so das ein Heizstab alle PV Leistung bis -20w verbraucht, kann ich open DTU on Battery so einstellen das der Wechselrichter aus bleibt und die Battery lädt solange genug Leistung von der größeren Pv kommt ? Ich hatte heute den fall das der geregelte Wechselrichter dann quasi auch die Leistung in den Heizstab gesteckt hat . Ist jetzt zwar nicht schlimm allerdings hätte ich lieber eine volle Batterie. 

Und eine Frage zum entladen der Batterie hätte ich noch, wann wir die Batterie entladen wenn es auf "Leeren, wenn voll" steht ?

Und ist es so das bei "Leeren zur Nacht" auch eine halb volle Batterie entladen wird ?

Grüße

**Comment by schlimmchen** [comment]
> Und eine Frage zum entladen der Batterie hätte ich noch, wann wir die Batterie entladen wenn es auf "Leeren, wenn voll" steht ?
> Und ist es so das bei "Leeren zur Nacht" auch eine halb volle Batterie entladen wird ?

Bitte schau dir https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Dynamic-Power-Limiter-%E2%80%90-Solar-Passthrough an. Wenn du Englisch nicht gut verstehst, dann nutze bitte ein automatisches Übersetzungswerkzeug, dann sollte trotzdem noch klar werden, was du wissen musst.

> kann ich open DTU on Battery so einstellen das der Wechselrichter aus bleibt und die Battery lädt solange genug Leistung von der größeren Pv kommt ?

Nein, da musst du schon irgendwie nachhelfen und die passende Entscheidungshilfe für den DPL liefern. Der kann ja nicht wissen, ob da ein Heizstab **überschüssige** Energie verbraucht, oder ob die Energie eigentlich benötigt wird um den Haushaltsbedarf zu decken. Da regeln dann ja offenbar zwei Instanzen gegeneinander. Die müssen dann irgendwie miteinander sprechen. Das musst du selbst irgendwie sicherstellen.

Du kannst den DPL über MQTT in einen anderen Modus als  den "normalen" einstellen, vielleicht hilft dir das: https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/MQTT#power-limiter-topics

**Comment by Michel83** [comment]
Vielen Dank für die Antwort.

Die Reglung funktioniert mittlerweile so wie sie soll ein wenig rum probieren aber jetzt scheint es zu Funktionieren.

Das mit dem DPL ist ein mega Tipp da ich diesen auf zwei schalten kann wenn ich unser Auto lade, danke für die Info.

Wo ich mir noch nicht ganz sicher bin sind die eingestellten Spannungen der Batterie, so richtig lesen kann ich das Datenblatt leider nicht:

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/36561525/b85cdfcd-b5cb-41a4-b45d-5c7f5b28e639)

von denen habe ich 3 Stück also 330ah mit 12v

Eingestellt habe ich:

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/36561525/edafe49e-dbf2-477f-a8f0-6dc8ea780fbe)


Denke Kaputt gehen kann so erst mal nix :-)

Grüße
